src/model/document.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2022 spdx contributors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from datetime import datetime
+from typing import List, Optional
+
+from src.model.actor import Actor
+from src.model.annotation import Annotation
+from src.model.external_document_ref import ExternalDocumentRef
+from src.model.extracted_licensing_info import ExtractedLicensingInfo
+from src.model.file import File
+from src.model.package import Package
+from src.model.relationship import Relationship
+from src.model.snippet import Snippet
+from src.model.version import Version
+
+
+class CreationInfo:
+    creators: List[Actor]  # min. 1 element
+    created: datetime
+    comment: Optional[str]
+
+    def __init__(self, creators: List[Actor], created: datetime, comment: Optional[str] = None):
+        self.creators = creators
+        self.created = created
+        self.comment = comment
+
+
+class Document:
+    data_license = "CC0-1.0"
```

Comment:
We might want to keep the data license field in the constructor. Else, when we parse an SPDX document with a different data license (which is not allowed according to spec, but nevertheless) we just overwrite it.

Reply:
Hm... the way we treat this for other properties would be an enum and a parsing error if an invalid value is encountered. Would you prefer that here?

Reply:
Well, it is still a license and for example [rdf handles it as one](https://spdx.github.io/spdx-spec/v2.3/document-creation-information/#62-data-license-field).

Reply:
Using rdf undermines your argument, I'm afraid :stuck_out_tongue: (unless I misunderstand the rdf ontology)
https://spdx.org/rdf/ontology/spdx-2-3/#d4e185

Reply:
Hmm, having never having a good look at it, I can't really argue here. Maybe we will find something yet to appease everyone :)